backend/src/app.service.ts

```diff
@@ -2,14 +2,27 @@ import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { Messages } from './db_interactions_modules/messages/messages.entity';
+import { User } from './db_interactions_modules/users/user.entity';
+import { Channel } from './db_interactions_modules/channels/channel.entity';
  
 @Injectable()
 export class AppService {
  constructor(
    @InjectRepository(Messages) private messagesRepository: Repository<Messages>,
+   @InjectRepository(User)private userRepository: Repository<User>,
+   @InjectRepository(Channel)private channelRepository: Repository<Channel>
  ) {}
- async createMessage(Messages: Messages): Promise<Messages> {
-   return await this.messagesRepository.save(Messages);
+ async createMessage(Messages: any){
```

Comment:
Messages is the parameter right?

It has the same name as the class Messages which can be confusing.
Also we are just creating one message, right?
```suggestion
 async createMessage(message: any){
```

Reply:
I will change this to msg_payload to avoid confusion in a future PR